libr/bin/format/elf/elf.c

```diff
@@ -3747,6 +3747,7 @@ static void _store_bin_sections(ELFOBJ *eo, const RVector *elf_bin_sections) {
 		ptr->type = elf_section_type_tostring (section->type);
 		ptr->add = !eo->phdr; // Load sections if there is no PHDR
 		ptr->perm = elf_flags_to_section_perms (section->flags);
+		ptr->flags = section->flags;
```

Comment:
same here, i think we can fix the bug without adding this field for now

Reply:
Done